rust/hyperlane-core/src/types/checkpoint.rs

```diff
@@ -10,7 +10,7 @@ use crate::{utils::domain_hash, Signable, Signature, SignedType, H160, H256};
 #[derive(Copy, Clone, Eq, PartialEq, Serialize, Deserialize, Debug)]
 pub struct Checkpoint {
     /// The mailbox address
-    pub mailbox_address: H256,
+    pub merkle_tree_hook_address: H256,
     /// The mailbox chain
     pub mailbox_domain: u32,
```

Comment:
I think i called this out in a diff comment but we should rename & change the `///` comments